Source/WebCore/loader/ApplicationManifestLoader.h

```diff
@@ -54,7 +54,7 @@ WTF_MAKE_NONCOPYABLE(ApplicationManifestLoader); WTF_MAKE_FAST_ALLOCATED;
 private:
     void notifyFinished(CachedResource&, const NetworkLoadMetrics&);
 
-    DocumentLoader& m_documentLoader;
+    Ref<DocumentLoader> m_documentLoader;
```

Comment:
This looks like this will introduce a memory leak.
DocumentLoader owns this object via a unique_ptr and now this object keeps a strong ref to its owner.

This means that once DocumentLoader has initialized DocumentLoader::m_applicationManifestLoader, it will never be deletable.